### ISSUE TITLE
Parallelization

### ISSUE THREAD

**Issue by tkram01** [cfusting/fast-symbolic-regression#4]
Do you have any pointers on where to look into the code to parallelize it? I would like to run a large job over many CPUs.

**Comment by cfusting** [comment]
Assuming it takes a bit of time to evaluate an individual you can parallelize the call to evaluate in https://github.com/cfusting/fastgp/blob/master/fastgp/algorithms/afpo.py over the population.

**Comment by skriegman** [comment]
yeah it's not really fast-symbolic-regression if it;s not in parallel
